apps/docs/package.json

```diff
@@ -13,10 +13,7 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "ci": "npm run lint && npm run format:diff",
-    "lint": "eslint --cache \"**/*.js\" && stylelint \"**/*.css\"",
-    "format": "prettier --config .prettierrc --write \"**/*.{js,jsx,ts,tsx,md,mdx}\"",
-    "format:diff": "prettier --config .prettierrc --list-different \"**/*.{js,jsx,ts,tsx,md,mdx}\""
+    "lint": "eslint --cache \"**/*.js\" && stylelint \"**/*.css\""
```

Comment:
We never run this, but maybe we should? Hook up any relevant tasks to this pacakge's `test` script, so it gets run when `npm test` is run from the monorepo root inc in CI.